test/skimOutputter.cpp

```diff
@@ -248,6 +250,8 @@ int main (int argc, char** argv)
 
   "VBFjj_mass_jetupTot","VBFjj_mass_jetdownTot",                       // VBFjj mass JES Total
 
+  "addJetCentr*","addJetForw*",                                        // Additional jets
```

Comment:
are these branches in smallTree? otherwise the skimOutputter will crash with ntuples produced with the original skimmer

Reply:
I tested it on an old ntuple and it does not crash because it's just copying the branches to the new ntuple, it just complains that it can't fin these branches.
But you raise a good point here because once we actually start using the additional jets it will probably crash.
I will create another skimOutputter_HHBtag that works with additional jets, do you think it's ok?